packages/flutter/lib/src/material/chip.dart

```diff
@@ -378,7 +378,7 @@ class _RenderChip extends RenderBox {
 
   // Set this to true to have outlines of the tap targets drawn over
   // the chip.  This should never be checked in while set to 'true'.
-  static const bool _debugShowTapTargetOutlines = false;
+  static const bool _debugShowTapTargetOutlines = true;
```

Comment:
we can just remove this flag entirely.

Reply:
No, because I still use it below.  Using the debugPaintSizeEnabled flag causes the display to be far too noisy to see the tap targets anymore.